.prow.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+
+# Only these tests work for csi-drivers-flex, E2E and sanity testing
+# will require further work.
+: ${CSI_PROW_TESTS:="unit serial parallel serial-alpha parallel-alpha"}
+
+# Customize deployment and E2E testing.
+: ${CSI_PROW_DEPLOY_SCRIPT:="deploy.sh"}
+: ${CSI_PROW_DEPLOYMENT:="kubernetes"}
+: ${CSI_PROW_E2E_TEST_PREFIX:="CSI Volumes"}
+
+. release-tools/prow.sh
+
+# Install custom E2E test suite as bin/tests.
+install_e2e () {
+    if [ -e "${CSI_PROW_WORK}/e2e.test" ]; then
+        return
+    fi
+
+    make builds-tests
```

Comment:
typo, should be `make build-tests`

Reply:
Huh, how the heck did that work?

Ah, missing error checking combined with building `bin/tests` earlier. Fixed and force-pushed.